AI/StupidAI/StupidAI.cpp

```diff
@@ -49,7 +49,8 @@ struct EnemyInfo
 	const CStack * s;
 	int adi, adr;
 	std::vector<BattleHex> attackFrom; //for melee fight
-	EnemyInfo(const CStack * _s) : s(_s), adi(0), adr(0)
+	EnemyInfo(const CStack * _s)
+		: s(_s), adi(0), adr(0)

```

Comment:
For "really may parameters" one member per line would look much better. Extra bonus - keep commas before member name, e.g.
```
EnemyInfo(const CStack * _s)
: s(_s)
, adi(0)
, adr(0)
```
Or even add vertical alignment (although I doubt that auto-formatters can do this)
```
EnemyInfo (const CStack * _s)
: s       (_s)
, adi     (0)
, adr     (0)
```
Such format helps a lot in readability for classes with 10+ members with names of different length